Pipfile

```diff
@@ -1,17 +1,31 @@
 [[source]]
-name = "pypi"
 url = "https://pypi.org/simple"
 verify_ssl = true
-
-[dev-packages]
-pytest = "~=6.2.4"
+name = "pypi"
 
 [packages]
+# linkml-runtime = "~=1.0, >=1.0.3" added by migration to linkml model template
+#   but version from cancerDHC required. For example "Direct matches"
 linkml = {editable = true, ref = "ccdh-dev", git = "https://github.com/cancerDHC/linkml.git"}
-mkdocs = "*"
+rdflib-pyld-compat = "*" # resinstating after removal by migration to linkml model template
+
+[dev-packages]
+# pytest = "~=6.2.4" removed by migration to linkml model template
+#   "replaced" with tox?
+tox = "*"
+twine = "*" # should this go in [dev-packages] or even make-venv/Pipfile ?
+# mkdocs = "*" removed by migration to linkml model template
+#   do mkdocs-material and mike provide the required functionality?
 mkdocs-material = "*"
 mike = "*"
-rdflib-pyld-compat = "*"
+
+[pipenv]
+# allow_prereleases = true added by migration to linkml model template
+allow_prereleases = true
 
 [requires]
 python_version = "3.8"
```

Comment:
Agreed. There are several slightly mismatched references to Python version here and elsewhere. Let's pin this down. I just don't really know what the required version really is!

Reply:
I trust Sujay regarding `MAINTAINERS.md` versus `CONTRIBUTION.md`

@gaurav ?

Reply:
I think we should be good with Python 3.7+?

Reply:
> In the [setup.py](https://github.com/cancerDHC/ccdhmodel/blob/issue-69-pypi/setup.py) file on this branch, you require Python 3.7+, but here you're requiring Python version 3.8 explicitly?
> 
> One of the ways to resolve this would be to either omit this `[requires]` section altogether, or change it to `python_version = "3"` which would mean the program can use any version of Python 3.

Sadly, it doesn't look like we can specify `~= 3.7` or something (according to the [docs](https://pipenv-fork.readthedocs.io/en/latest/basics.html#specifying-versions-of-python)), which would be ideal. I think `python_version = "3"` is probably the next best thing: we can let people know that is a Python 3 program without locking them in to a particular version of Python. We should eventually test against multiple versions of Python once we have automated testing of this repo working -- see https://github.com/cancerDHC/ccdhmodel/issues/12#issuecomment-903335010.

> I trust Sujay regarding `MAINTAINERS.md` versus `CONTRIBUTION.md`
> 
> @gaurav ?

Based on the [example from the Atom repo](https://github.com/atom/atom/blob/master/CONTRIBUTING.md) anyway, it looks like CONTRIBUTING.md should be information on contributing to CRDCH development, i.e. let people know where issues should be filed, that we welcome PRs for review, and with some information on how the repository is laid out and where changes should be made (there's a nice example [in this Gist](https://gist.github.com/PurpleBooth/b24679402957c63ec426)). It looks like the build information usually goes in [`BUILD.md`](https://github.com/hashcat/hashcat/blob/master/BUILD.md) or [`BUILDING.md`](https://github.com/nodejs/node/blob/master/BUILDING.md). For this PR, however, whatever is quickest is probably best for now -- we can open an issue for this and polish it up in the next quarter. I agree that one of those would be better than MAINTAINERS.md, which AFAICT usually lists the maintainers of this package, which we list in AUTHORS.md instead.

(This definitely sounds like something where, once we figure out what we do, we should create a PR against the LinkML template repo so that future users don't get confused about this as we are here 😁 )

Of course, eventually we'd like the build instructions to be nowhere at all -- they should be written in the Makefile or setup.py, and README.md should have the instructions on how to run and configure them. These instructions should be exercised by a GitHub Action, whose logs we link to from README.md, so it's clear what it looks like when this repository works correctly. But that's probably a ways away for now :)